docs/spec.json

```diff
@@ -0,0 +1,383 @@
+{
+  "name": "corectl",
+  "info": {
+    "title": "Specification for corectl",
+    "description": "corectl contains various commands to interact with the Qlik Associative Engine. See respective command for more information",
+    "version": "v0.4.0",
+    "license": "MIT"
+  },
+  "clispec": "0.1.0",
+  "description": "corectl contains various commands to interact with the Qlik Associative Engine. See respective command for more information",
+  "x-qlik-stability": "experimental",
+  "flags": {
+    "app": {
+      "alias": "a",
+      "description": "App name, if no app is specified a session app is used instead."
+    },
+    "bash": {
+      "description": "Bash flag used to adapt output to bash completion format",
+      "default": "false"
```

Comment:
Should the flag have a parameter/argument of type bool with a default value of false? The spec do not currently have default on the flag itself but on the parameter/argument

Reply:
yes - yes it should 😄golang doesn't have undefined. either true or false